crates/vm/src/expr.rs

```diff
@@ -1436,14 +1446,24 @@ impl QueryExpr {
                 self.query.push(Query::IndexScan(IndexScan { table, columns, bounds }));
                 self
             }
-            // merge with a preceding lower bounded index scan (inclusive)
+            // merge with a preceding lower bounded index scan (enclusive)
             Query::IndexScan(IndexScan {
```

Comment:
This word [exist](https://wikidiff.com/enclusive/inclusive)?

Reply:
🤦. I replaced one letter but not the other when fixing a copy-paste error. Pushed a fix.